proxy/http2/Http2ConnectionState.cc

```diff
@@ -1187,7 +1187,9 @@ Http2ConnectionState::release_stream(Http2Stream *stream)
       if (fini_received) {
         // We were shutting down, go ahead and terminate the session
         ua_session->destroy();
-        ua_session = nullptr;
+        // Can't do this because we just destroyed right here ^,
+        // or we can use a local variable to do it.
+        // ua_session = nullptr;
```

Comment:
hmm, this line sets a local variable to nullptr. should be fine.

Reply:
I agree with @zizhong, this is an invalid change..

Reply:
No, ua_session is a member variable of Http2ConnectionState which was just freed in the previous destroy call.